File: Builder/Cocina.java
public class Cocina {
    private PizzaBuilder pizzaBuilder;

    public void setPizzaBuilder(PizzaBuilder pb)
    {
        pizzaBuilder = pb;
    }

    public Pizza getPizza()
    {
        return  pizzaBuilder.getPizza();
    }

    public void construirPizza()
    {
        pizzaBuilder.crearNuevaPizza();
        pizzaBuilder.buildMasa();
        pizzaBuilder.buildRelleno();
        pizzaBuilder.buildSalsa();
    }

}

File: Builder/Python/Cocina.py
from PizzaBuilder import *
class Cocina:

    _pizzaBuilder = None

    def setPizzaBuilder(self, pb):
        self._pizzaBuilder = pb
    
    def getPizza(self):
        return self._pizzaBuilder.getPizza()
    
    def constuirPizza(self):
        self._pizzaBuilder.crearNuevaPizza()
        self._pizzaBuilder.buildMasa()
        self._pizzaBuilder.buildRelleno()
        self._pizzaBuilder.buildSalsa()  